notebooks/nyc_taxi_years_correlation.ipynb

```diff
@@ -58,9 +58,9 @@
     }
    ],
    "source": [
-    "!if [ ! -f \"tzones_lonlat.json\" ]; then curl \"https://data.cityofnewyork.us/api/geospatial/d3c5-ddgc?method=export&format=GeoJSON\" -o tzones_lonlat.json; else echo \"tzones_lonlat.json found\"; fi\n",
-    "!if [ ! -f \"taxi2016.csv\" ]; then curl https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2016-01.csv -o taxi2016.csv; else echo \"taxi2016.csv found\"; fi   \n",
-    "!if [ ! -f \"taxi2017.csv\" ]; then curl https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2017-01.csv -o taxi2017.csv; else echo \"taxi2017.csv found\"; fi"
+    "!if [ ! -f \"tzones_lonlat.json\" ]; then curl 'https://data.cityofnewyork.us/api/geospatial/d3c5-ddgc?method=export&format=GeoJSON' -o tzones_lonlat.json; else echo \"tzones_lonlat.json found\"; fi\n",
+    "!if [ ! -f \"taxi2016.csv\" ]; then curl 'https://storage.googleapis.com/anaconda-public-data/nyc-taxi/csv/2016/yellow_tripdata_2016-01.csv' -o taxi2016.csv; else echo \"taxi2016.csv found\"; fi\n",
+    "!if [ ! -f \"taxi2017.csv\" ]; then curl 'https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2017-01.parquet' -o taxi2017.csv; else echo \"taxi2017.csv found\"; fi"
```

Comment:
!if [ ! -f \"taxi2017.csv\" ]; then curl 'https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2017-01.parquet' -o taxi2017.csv; else echo \"taxi2017.csv found\"; fi

should be 
!if [ ! -f \"taxi2017.csv\" ]; then curl 'https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2017-01.parquet' -o taxi2017.parquet; else echo \"taxi2017.parquet found\"; fi